examples/bicluster/plot_bicluster_newsgroups.py

```diff
@@ -68,7 +68,7 @@ def build_tokenizer(self):
 newsgroups = fetch_20newsgroups(categories=categories)
 y_true = newsgroups.target
 
-vectorizer = NumberNormalizingVectorizer(stop_words='english', min_df=5)
+vectorizer = NumberNormalizingVectorizer(max_df=0.4, min_df=5)
```

Comment:
How do you choose the value of max_df here? Is it possible to gain better result with another max_df?

Reply:
See https://github.com/scikit-learn/scikit-learn/issues/10834#issuecomment-375241811

Reply:
Chosen after testing out locally. With current max_df = 0.4  there were optimum`v_measure_score`. Please do let me know if I am to tweak it further. Thanks!